x/crisis/keeper.go

```diff
@@ -0,0 +1,36 @@
+package crisis
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// Keeper - crisis keeper
+type Keeper struct {
+	routes     []InvarRoute
```

Comment:
Is there a reason we are not using something simillar with `baseapp.Router` here?

Reply:
because of how the crisis.routes are currently used, I don't think they warrant the complexity of adding it to a special interface like baseapp.Routes has. Also baseapp.Routes doesn't have an easy iterate all functionality which is used here.

Reply:
We're going to have a generic router soon anyway #3976.